CHANGELOG.md

```diff
@@ -3,6 +3,13 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+v0.35.0
+------------------------------
+*August 29, 2018*
+
+### Changed
+- Added pseudo-locale `qps-ploc` for localisation testing on Windows.
```

Comment:
It does - see the link to the docs in the description 😃

Reply:
https://docs.microsoft.com/en-us/windows/desktop/intl/pseudo-locales#pseudo-locale-names-and-identifiers:

> The pseudo-locales have locale names that are chosen from the private use space to avoid conflict with possible strings introduced into the International Organization for Standardization (ISO) 639 and ISO 3166 standards. Each pseudo-locale also has its own locale identifier.